bin/generate_key.ts

```diff
@@ -1,9 +1,8 @@
+require("dotenv").config()
```

Comment:
What's the idea here? To get this before `crypto` is required? I think it's a bit scary because `import` is supposed to come before any application code and the specific way we are compiling typescript might be the only reason this is allowed.

I think you can actually do `import 'dotenv/config'` according to the docs (https://www.npmjs.com/package/dotenv) to get around this.

Reply:
oh that's from the original PR as well. deleting that